std/http/file_server.ts

```diff
@@ -18,16 +18,18 @@ interface EntryInfo {
   name: string;
 }
 
-interface FileServerArgs {
+export interface FileServerArgs {
   _: string[];
   // -p --port
-  p: number;
-  port: number;
+  p?: number;
+  port?: number;
   // --cors
-  cors: boolean;
+  cors?: boolean;
+  // -d
+  d?: true | "false";
```

Comment:
why not boolean?

Reply:
Because when the `parse` module parses `-d false`, it actually gets `{ d: "false" }`.
Set `true` because we want it to be always enabled by default...

Reply:
@ry `-d` is meaningless to accept parameters other than the string "false", so I now think we can directly set to "false".